tls/s2n_tls13_handshake.c

```diff
@@ -317,6 +317,19 @@ static int s2n_tls13_handle_master_secret(struct s2n_connection *conn)
     return S2N_SUCCESS;
 }
 
+static int s2n_tls13_handle_resumption_master_secret(struct s2n_connection *conn)
+{
```

Comment:
Why can't this function be unit tested? What is the blocker?

Reply:
Given arbitrary inputs, we don't know what the output should be :) The known value tests at least give us the output for a specific set of inputs.

Reply:
I think we would test this in a self-talk test. Otherwise we can't check that the resumption_master_secret was generated correctly.